lib/matplotlib/tests/test_backend_ps.py

```diff
@@ -134,6 +134,46 @@ def test_patheffects():
             fig.savefig(ps, format='ps')
 
 
+@cleanup
+@needs_tex
+@needs_ghostscript
+def test_tilde_in_tempfilename():
+    # Tilde ~ in the tempdir path (e.g. TMPDIR, TMP oder TEMP on windows
+    # when the username is very long and windows uses a short name) breaks
+    # latex before https://github.com/matplotlib/matplotlib/pull/5928
+    import tempfile
+    import shutil
+    import os
+    import os.path
+
+    dirpath = None
```

Comment:
yes, that should have been `base_tempdir`, so that if that is set, it will remove in the finally...